unfoldingWord/translationCore
# Only allow Titus projects in non-developer mode

#### This pull request addresses:
- Only allow Titus projects in non-developer mode 

#### How to test this pull request:
- Make sure to disable developer mode.
- Try loading a non-Titus project.
- Should see an alert saying "You can only load Titus projects for now.".


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2239)
